File: validate Sequence/O(n)_time and_O(1)_space.js
function isValidSubsequence(array, sequence) {
  // Write your code here.
  let seqInd = 0;
  for (let i of array) {
    if (seqInd === sequence.length) break;
    if (i === sequence[seqInd]) seqInd++;
  }
  return seqInd === sequence.length;
}

function isValidSubsequence1(array, sequence) {
    // Write your code here.
    let arrInd=0
    let seqInd=0
    while(arrInd < array.length && seqInd < sequence.length){
      if (array[arrInd] === sequence[seqInd]) seqInd++;
      arrInd++;
    }
    return seqInd===sequence.length
  }

File: validate Sequence/O(n)_time_and_O(1)_space.py
def isValidSubsequence(array, sequence):
    # Write your code here.
    seqInd = 0
    for i in array:
        if seqInd == len(sequence):
            break
        if i == sequence[seqInd]:
            seqInd +=1
    return seqInd == len(sequence)

def isValidSubsequence1(array, sequence):
    # Write your code here.
    arrInd=0
    seqInd=0
    while arrInd < len(array) and seqInd < len(sequence):
        if array[arrInd] == sequence[seqInd]:
            seqInd += 1
        arrInd+= 1
    return seqInd == len(sequence)